docs/user_guide/representing/first_step.md

```diff
@@ -0,0 +1,135 @@
+# Representing
+
+At the heart of `DocArray` lies the concept of [`BaseDoc`][docarray.base_doc.doc.BaseDoc].
+
+A [BaseDoc][docarray.base_doc.doc.BaseDoc] is very similar to a [Pydantic](https://docs.pydantic.dev/)
+[`BaseModel`](https://docs.Pydantic.dev/usage/models) - in fact it _is_ a specialized Pydantic `BaseModel`. It allows you to define custom `Document` schemas (or `Model` in
+the Pydantic world) to represent your data.
+
+## Basic `Doc` usage.
+
+Before going into detail about what we can do with [BaseDoc][docarray.base_doc.doc.BaseDoc] and how to use it, let's
+see what it looks like in practice.
+
+The following Python code defines a `BannerDoc` class that can be used to represent the data of a website banner.
+
+```python
+from docarray import BaseDoc
+from docarray.typing import ImageUrl
+
+
+class BannerDoc(BaseDoc):
+    image_url: ImageUrl
+    title: str
+    description: str
+```
+
+You can then instantiate a `BannerDoc` object and access its attributes.
+
+```python
+banner = BannerDoc(
+    image_url='https://example.com/image.png',
+    title='Hello World',
+    description='This is a banner',
+)
+
+assert banner.image_url == 'https://example.com/image.png'
+assert banner.title == 'Hello World'
+assert banner.description == 'This is a banner'
+```
+
+## `BaseDoc` is a Pydantic `BaseModel`
+
+The class [BaseDoc][docarray.base_doc.doc.BaseDoc] inherits from Pydantic [BaseModel](https://docs.pydantic.dev/usage/models). So you can use
+all the features of `BaseModel` in your `Doc` class. 
+
+This means that `BaseDoc`:
+
+* Will perform data validation: `BaseDoc` will check that the data you pass to it is valid. If not, it will raise an 
+error. Data being "valid" is actually defined by the type used in the type hint itself, but we will come back to this concept later. (TODO add typing section)
+* Can be configured using a nested `Config` class, see Pydantic [documentation](https://docs.pydantic.dev/usage/model_config/) for more detail on what kind of config pydantic offers.
+* Can be used as a drop-in replacement for `BaseModel` in your code and is compatible with tools that use Pydantic like [FastAPI]('https://fastapi.tiangolo.com/').
```

Comment:
Just a note to reference FastAPI integration page here that I'm currently working on whoever merges the second